cal/cal/templates/home_logged_in.html

```diff
@@ -155,11 +156,12 @@ <h4 class="well">Tags - Cumulative</h4>
     {% if request.user.is_staff %}
       <div>
         <h4 class="well">Options</h4>
-        <a href="/v1/sync"><button class="btn btn-primary pull-left">Sync</button></a>
-        <a href="/v1/sync?full_sync=true"><button class="btn btn-primary pull-left">Full Sync</button></a>
-        <a href="/v1/sync?sync_all=true"><button class="btn btn-primary pull-left">Sync All Calendars</button></a>
+        <a href="/v1/sync"><button class="btn btn-primary pull-left">Inc. Sync Main Calendar</button></a>
+        <a href="/v1/sync?full_sync=true"><button class="btn btn-primary pull-left">Full Sync Main Calendar</button></a>
+        <a href="/v1/sync?sync_all=true"><button class="btn btn-primary pull-left">Inc. Sync All Calendars</button></a>
         <a href="/v1/sync?full_sync=true&sync_all=true"><button class="btn btn-primary pull-left">Full Sync All Calendars</button></a>
         <a href="/auth/clear"><button class="btn btn-primary pull-left">Reset Credentials</button></a>
+        <a href="/admin/generate-categories"><button class="btn btn-primary pull-left">Generate Categories</button></a>
```

Comment:
Added styles inline; hopefully you are okay with this since these buttons only appear for admins and I don't want to accidentally apply a style to something a normal user would see